com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1656,6 +1656,12 @@ internal void HandleApproval(ulong ownerClientId, bool createPlayerObject, uint?
                     var networkObject = SpawnManager.CreateLocalNetworkObject(false, playerPrefabHash ?? NetworkConfig.PlayerPrefab.GetComponent<NetworkObject>().GlobalObjectIdHash, ownerClientId, null, position, rotation);
                     SpawnManager.SpawnNetworkObjectLocally(networkObject, SpawnManager.GetNetworkObjectId(), false, true, ownerClientId, false);
 
+                    var networkObjectChildren = networkObject.GetComponentsInChildren<NetworkObject>();
```

Comment:
I wonder what would happen if we try to spawn a NetworkPrefab with nested NetworkObjects in it. it should also fail too, right? if so, can we catch that too in the NetworkPrefab spawning flow and Debug.LogError there too?

Reply:
It fails in the same way. The object is present in the hierarchy but the NetworkObject is not spawned and the network functionality is broken. I'll add the same check, verify it errors out and update this PR. Thanks!

Reply:
Updated. Please consider re-reviewing :-)